plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs

```diff
@@ -57,6 +61,16 @@ schnorrSecp256k1Prop = do
   cover 18 "happy path" . is (_Shouldn'tError . _AllGood) $ testCase
   runTestDataWith testCase id VerifySchnorrSecp256k1Signature
 
+ed25519Prop :: PropertyT IO ()
```

Comment:
The Ed25519 functions are identical to the Scnorr ones except for types.  In #4606 I have a [version of this file](https://github.com/input-output-hk/plutus/blob/33cc7760863379d8c14f7b1e285978b904f87df5/plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SignatureVerification.hs) which combines them into a single function generic over the algorithm.. I've undone that here because there's work in progress on the scep256k1 code which may lead to the tests being updated and I don't want to introduce conflicts with that.